test/builtin_dataset_mocks.py

```diff
@@ -61,27 +60,30 @@ def _parse_mock_info(self, mock_info):
 
         return mock_info
 
-    def prepare(self, home, config):
+    def prepare(self, home, **options):
```

Comment:
We're using the name `options` here but IIUC just above we call it `config`? Perhaps worth using a single name? (I wouldn't mind keeping config to minimize the changes)

Reply:
I think the plural is because it refers to argumentS (`kwargs`), but it doesn't shock me to pass `**some_dict` without a plural. I'd err towards consistency in the naming here - no pref between **options or **config